src/Controls/DualScreen/src/DualScreenInfo.shared.cs

```diff
@@ -150,7 +162,7 @@ Rectangle GetHingeBounds()
 		static DualScreenInfo OnCreate()
 		{
 			DualScreenInfo dualScreenInfo = new DualScreenInfo(null);
-			dualScreenInfo._twoPaneViewLayoutGuide.WatchForChanges();
+			dualScreenInfo._twoPaneViewLayoutGuide.WatchForChanges(); // HACK: no-op!
```

Comment:
If this is a no-op can we delete `WatchForChanges` ?

Reply:
Seems like it - commented out for removal during refactor